<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eliminar_estudiante</class>
 <widget class="QWidget" name="Eliminar_estudiante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>161</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>161</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Eliminar Estudiante</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="lay_ver_principal">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbl_eliminar_estudiante">
       <property name="styleSheet">
        <string notr="true">font: 35pt &quot;Gill Sans MT Condensed&quot;;</string>
       </property>
       <property name="text">
        <string>Eliminar estudiante</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="lay_frm_datos">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_codigo">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Gill Sans MT&quot;;</string>
         </property>
         <property name="text">
          <string>ID/Codigo: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_codigo">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(0, 0, 0, 255);
padding-bottom: 5px;
font: 10pt &quot;Gill Sans MT&quot;;</string>
         </property>
         <property name="placeholderText">
          <string>Inserte ID/codigo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_eliminar_estudiante">
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_eliminar_estudiante {
	background-color: rgba(255,175,15,156);
	border-radius: 5px;
	font: 15pt &quot;Gill Sans MT Condensed&quot;;
}

QPushButton#btn_eliminar_estudiante:hover {
	background-color: rgba(255,200,15,156);
	border-radius: 5px;
	
	font: 15pt &quot;Gill Sans MT Condensed&quot;;
}

QPushButton#btn_eliminar_estudiante:pressed {
	background-color: rgba(115,0,15,115);
	border-radius: 5px;
	
	font: 15pt &quot;Gill Sans MT Condensed&quot;;
}</string>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
